docs/index.md

```diff
@@ -1175,6 +1175,15 @@ Mocha allows you to define custom reporters. For more information see the [wiki]
 
 **The HTML reporter is not intended for use on the command-line.**
 
+### Printing error messages immediately
+
+If you have long-running tests you may want to print errors immediately to the console so that you don't have to wait for the tests to finish to see why a given test failed.
+The Spec, Dot Matrix, List, and Min reporters support a `showErrorsImmediately` option that will output error details immediately when they occur instead of after all tests have finished:
```

Comment:
please refer to the reporters in preformatted text (and lowercase), so: `spec`, `dot`, `list`, and `min`.  also this can be simplified to something like:

> The `spec`, `dot`, `list`, and `min` reporters support a `showErrorsImmediately` option which will print errors as they occur: